src/layouts/_proxied-dot-io/boundary/index.tsx

```diff
@@ -49,13 +56,59 @@ function BoundaryIoLayout({
 						hideOnMobile
 					/>
 				)}
-				<HashiStackMenu onPanelChange={() => null} />
-				<ProductSubnav />
+				<ProductSubnav
+					menuItems={[
+						{
+							text: 'Overview',
+							url: '/',
+							type: 'inbound',
+						},
+						'divider',
+						useCaseNavItems.length > 0
+							? {
+									text: 'Use Cases',
+									submenu: [
+										...useCaseNavItems.map((item) => {
```

Comment:
I don't think you need the spread operator here. Instead, it'd be something like:

```ts
submenu: useCaseNavItems.map((item) => {
```

Reply:
https://github.com/hashicorp/dev-portal/pull/526/commits/a80d00f526dc25c3f090a01dde16ecca9222e1f8